### PR TITLE
updated wall-spacer and color of preview

### PR DESCRIPTION
**Pull request by hoergen**

### CODE REVIEW

File: view/theme/frio/scheme/plusminus.css [hunk 1]
```diff
@@ -96,4 +96,9 @@ aside .widget, .form-control, .panel, .nav-container, .wall-item-content, .e-con
     margin-bottom: 0px;
     border:0px;
     padding:0px;
+	color:black;
```
Comment (MrPetovan) [inline view/theme/frio/scheme/plusminus.css]: Code standards: Wrong indentation, please use the surrounding indentation.

File: view/theme/frio/css/style.css [hunk 2]
```diff
@@ -1646,6 +1646,11 @@ section #jotOpen {
     z-index: 1000;
 }
 
+/* Space between content and head */
+.wall-spacer {
+	height: 10px;
```
Comment (MrPetovan) [inline view/theme/frio/css/style.css]: Code standards: Wrong indentation, please use the surrounding indentation.

Comment (hoergen) [reply view/theme/frio/css/style.css]: Hmm looked fine, when i edited it. Corrected it now. Should be fine now.

Comment (MrPetovan) [reply view/theme/frio/css/style.css]: This still is a tab when indentation is 4 spaces around.

Comment (hoergen) [reply view/theme/frio/css/style.css]: I used spaces .. hmm somethings wrong

**Comment by hoergen** [comment]
changed it again. Hope this time its correct?

File: view/theme/frio/css/style.css (hunk 2 above)
Comment (MrPetovan) [reply view/theme/frio/css/style.css]: Nope, still a tab.

**Comment by MrPetovan** [comment]
Nope, the specific line still has a tab indentation.

**Comment by hoergen** [comment]
I should look in the right file ;)) NOW it should be ok ;) Thanks for being so patient ;)

**Comment by hoergen** [comment]
Conflict solved

File: view/theme/frio/css/style.css [hunk 3]
```diff
@@ -1668,7 +1668,8 @@ section #jotOpen {
 
 /* Space between content and head */
 .wall-spacer {
-	height: 10px;
+	  height: 10px;
```
Comment (MrPetovan) [inline view/theme/frio/css/style.css]: And we're back to a wrong indentation, this time there's a tab and two spaces.

**Comment by hoergen** [comment]
After editing and commiting it looks good

**Comment by hoergen** [comment]
YAAAAYYY 🎊🎉🎶🎆🤘

**Pull request merged**